### ISSUE TITLE
Tooltip example fails WCAG SC 1.4.13 Content on Hover or Focus

### ISSUE THREAD

**Issue by andypillip** [mdn/content#24244]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

### What specific section or headline is this issue about?

Description and example

### What information was incorrect, unhelpful, or incomplete?

The SC requires the tooltip to stay open, if the mouse is moved over it (hovered).

This is not mentioned in the the description’s text, and the example does not take this into account.

### What did you expect to see?

Some addition that the tooltip must stay open if hovered, and either a warning along with the example, or a solution for this.

### Do you have any supporting links, references, or citations?

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html#hoverable

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tooltip_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tooltip_role/index.md
* Last commit: https://github.com/mdn/content/commit/7d2071f78f38f8be38888a32c599aebba348941b
* Document last modified: 2022-09-13T12:22:39.000Z

</details>

**Comment by andypillip** [comment]
Further, why is there a section about the _accessible name_ for a tooltip via `aria-label`? It’s neither required, nor does it seem very useful to me. The whole point of a tooltip is to make additional contents _visible_, is it not?

**Comment by estelle** [comment]
Updated with accessible name edits, but still needs work. Per https://github.com/mdn/content/pull/25587#issuecomment-1489260640 (summarizing here so people don't have to go back and forth). The example on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role needs to be updated

The page reads:

> It [the tooltip] should disappear automatically when the focus on the owning element is lost or the mouse is moved.

This isn't enough. It should NOT disappear if the mouse is moved from the owning element to the tooltip. In that case, the tooltip MUST stay open.

The SC reads:

> Persistent
> The additional content remains visible until the hover or focus trigger is removed, the user dismisses it, or its information is no longer valid.

This allows for classical tooltips, so they do not fail the criterion in general. The example does not satisfy the hoverable part:

> Hoverable
> If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;

If the tooltip does not support this, it fails the criterion, which is what is happening in the current example.
